curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/use-the-u-tag-to-underline-text.md

```diff
@@ -25,7 +25,7 @@ dashedName: use-the-u-tag-to-underline-text
 assert($('u').length === 1);
 ```
 
-`u` 标签的文本内容应为 `Ph.D. students`。
+`u` 标签的文本内容应为 `Ph.D students`。
```

Comment:
This is still an issue, because it was not translated correctly.  Basically, the text inside the `code` tag was changed from the English.
```suggestion
`u` 标签的文本内容应为 `Ph.D. students`。
```

Reply:
I was just looking at this on Crowdin and saw no issue - you must be too quick for me 😁